### PR TITLE
TCK fixes to make it run as a single dependency

### PR DESCRIPTION
**Pull request by radcortez**

### CODE REVIEW

File: tracing/tck/pom.xml [hunk 1]
```diff
@@ -88,7 +88,6 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
             <version>${version.awaitility}</version>
-            <scope>provided</scope>
```
Comment (radcortez) [inline tracing/tck/pom.xml:91]: If marked as provided, the consumer must also include the dependency.

File: tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/TestConfigSource.java [hunk 2]
```diff
@@ -1,46 +0,0 @@
...
- */
-package org.eclipse.microprofile.telemetry.tracing.tck;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
-
-import org.eclipse.microprofile.config.spi.ConfigSource;
-
-public class TestConfigSource implements ConfigSource {
```
Comment (radcortez) [inline tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/TestConfigSource.java:29]: This is not being used.

File: tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanTest.java [hunk 3]
```diff
@@ -24,10 +24,8 @@
 import static io.opentelemetry.api.trace.SpanKind.CLIENT;
 import static io.opentelemetry.api.trace.SpanKind.INTERNAL;
 import static io.opentelemetry.api.trace.SpanKind.SERVER;
-import static io.opentelemetry.semconv.trace.attributes.SemanticAttributes.HTTP_HOST;
```
Comment (radcortez) [inline tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanTest.java:27]: Deprecated in OTel 1.19.

File: tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanTest.java [hunk 4]
```diff
@@ -24,10 +24,8 @@
...
 import static io.opentelemetry.semconv.trace.attributes.SemanticAttributes.HTTP_METHOD;
 import static io.opentelemetry.semconv.trace.attributes.SemanticAttributes.HTTP_SCHEME;
-import static io.opentelemetry.semconv.trace.attributes.SemanticAttributes.HTTP_SERVER_NAME;
```
Comment (radcortez) [inline tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanTest.java:30]: Deprecated in OTel 1.19.

File: tracing/tck/src/test/resources/META-INF/microprofile-config.properties [hunk 5]
```diff
@@ -1,5 +0,0 @@
-otel.traces.exporter=in-memory
```
Comment (radcortez) [inline tracing/tck/src/test/resources/META-INF/microprofile-config.properties:1]: This is not being used.

**Comment by JanWesterkamp-iJUG** [comment]
@radcortez looks good to me.

But, before merging your PR, can we first get my PR https://github.com/eclipse/microprofile-telemetry/pull/60 reviewed and merged - there will be merge conflicts (again), because of the changes on the test classes...

Can you eventually review and merge it, if ok?
And afterwards merge your changes in - the Otel Java version will then be set to 1.19.0 too.

**Review by Emily-Jiang — APPROVED** [review]

**Comment by radcortez** [comment]
This can go after #60. But I need these changes to run the TCK in SmallRye OpenTelemetry smoothly.

**Review by brunobat — APPROVED** [review]

**Comment by Emily-Jiang** [comment]
@Azquelt can you create a separate issue with a proposed PR as this one has closed? We plan to do another RC with OTel 1.19 alignment asap.

**Comment by radcortez** [comment]
Yes, this can also be removed.

**Pull request merged**